tests/acceptance/helpers/path.js

```diff
@@ -1,8 +1,19 @@
 const join = require('join-path')
 const _ = require('lodash/fp')
+const assert = require('assert')
+const normalize = _.replace(/^\/+|$/g, '')
+const parts = _.pipe(normalize, _.split('/'))
+const relativeTo = function (basePath, childPath) {
```

Comment:
*Nitpicky*: Redundant naming in the variable (IMO, the context that it works with `path` is pretty well established here). I'm fine either way, though.
 
```suggestion
const relativeTo = function (base, child) {
```